t04_refactored.py

```diff
@@ -177,8 +177,51 @@ def team_8_adv(username, delay):
 
 def team_9_adv(username, delay):
     pass
-    # TODO Add your code here
+    """ the code asks where you want tp go but the only option is North, after choosing North it asks you what you're doing here and you choose options 1, 2 or 3. after choosing the option all options leads to turning into a frog at the end.
+        https://docs.google.com/document/d/1guMbODOgcKlr7DTSsVviL7zItomDFJwApO42bZtpuQc/edit?usp=sharing 
+        Din din
+        Kirsten
+    """
+    question = ""
+    delay = 1.0
+    direction = input("Where would you like to go, You can go North")
+
+    #this asks which option you would like to choose after writing North
+    if direction == "North":
+        print("It's a wizard!")
+        question = input("""What do you ask the wizard(1,2,3)?
+        1)What are you doing here? 
+        2)Can you help me?
+        3)Turn me into a frog\n: """)
+        sleep(delay)
+
+#option 1 teleports you into the wizard's cave, and they perform a spell on you
+    if question == "1":
```

Comment:
This `if` 'elif` `else` chain should be indented inside the `if direction ...` line. This is because `question` can only exist inside that conditional.